src/test/java/nl/tudelft/ewi/devhub/server/backend/DeliveriesBackendTest.java

```diff
@@ -0,0 +1,360 @@
+package nl.tudelft.ewi.devhub.server.backend;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Set;
+
+import nl.tudelft.ewi.devhub.server.database.controllers.Deliveries;
+import nl.tudelft.ewi.devhub.server.database.entities.Assignment;
+import nl.tudelft.ewi.devhub.server.database.entities.Course;
+import nl.tudelft.ewi.devhub.server.database.entities.Delivery;
+import nl.tudelft.ewi.devhub.server.database.entities.Delivery.Review;
+import nl.tudelft.ewi.devhub.server.database.entities.DeliveryAttachment;
+import nl.tudelft.ewi.devhub.server.database.entities.Group;
+import nl.tudelft.ewi.devhub.server.database.entities.User;
+import nl.tudelft.ewi.devhub.server.web.errors.ApiError;
+import nl.tudelft.ewi.devhub.server.web.errors.UnauthorizedException;
+
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Matchers;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DeliveriesBackendTest extends BackendTest {
+
+	private static final String fileName = "myFile.txt";
+
+	private static final String pathName = "my/fancy/path/";
```

Comment:
Path should be separated by the path separator used by the operating system. Therefore the test fails in Linux (CI) environments. Use: `"my/fancy/path/".replaceAll("/", File.separator)`.